pandas-stubs/_typing.pyi

```diff
@@ -166,7 +166,8 @@ class WriteExcelBuffer(WriteBuffer[bytes], Protocol):
 
 FilePath: TypeAlias = str | PathLike[str]
 
-Axis: TypeAlias = str | int
+AxisInt: TypeAlias = Literal[0, 1]
+Axis: TypeAlias = AxisInt | Literal["index", "columns", "rows"]
```

Comment:
@twoertwein @Dr-Irv While I understand the intention, this one causes a problem with the following types from Pandas:

`pandas.core.frame.DataFrame.reorder_levels(self, order: Sequence[Axis], axis: Axis = 0)`

The order parameter expects a sequence of ints or strings - with this change this is forbidden.

From my code:

```
reportquery.py:612: error: Argument 1 to
"reorder_levels" of "DataFrame" has incompatible type "Sequence[str]"; expected
"Sequence[Union[Literal[0, 1], Literal['index', 'columns', 'rows']]]"
[arg-type]
            df = df.reorder_levels(column_names, axis=1)
```

PS: Sorry for not creating an issue (yet). I am short on time.

Reply:
> PS: Sorry for not creating an issue (yet). I am short on time.

When you do have the time, please create an issue with a full code sample.  

This has picked up that `reorder_levels()` has an incorrect type for `order`.  It should be `Sequence[Hashable]`

Reply:
Here we go: https://github.com/pandas-dev/pandas-stubs/issues/560